projects/scion/toolkit/karma.conf.js

```diff
@@ -10,7 +10,11 @@
 
 // Karma configuration file, see link for more information
 // https://karma-runner.github.io/1.0/config/configuration-file.html
-process.env.CHROME_BIN = require('puppeteer').executablePath();
```

Comment:
See my comment above.

If we decide to still using Playwright as the browser provider, I think we do not need to configure `PLAYWRIGHT_BROWSERS_PATH ` since running `npm ci` and not `npm install` in the CI pipeline (GitHub Actions). Or did I misunderstand the documentation?

Reply:
As discussed, we'll continue to use `puppeteer`.